ld/devices.data

```diff
@@ -48,18 +48,18 @@
 # Example:
 #
 # --- devices.data file ---
-# stm32f05[01]?4* stm32f0 ROM=16K RAM=4K
-# stm32f0 stm32 ROM_OFF=0x08000000 RAM_OFF=0x20000000
-# stm32 END
+# stm32f103?b* stm32f1 ROM=128K RAM=20K
+# stm32f1 END ROM_OFF=0x08000000 RAM_OFF=0x20000000 CPU=cortex-m3 FPU=soft
 #
 # --- queried chip name ---
-# stm32f051c8t6
+# stm32f103rbt6
```

Comment:
It was correct. But I wanted to give a more general case. In the old example, `scripts/genlink.awk` only prints out something for MODE=DEFS. In the new example we will have a nonempty output for MODE=DEF|CPU|FPU.